minijinja/src/vm/mod.rs

```diff
@@ -484,23 +521,7 @@ impl<'env> Vm<'env> {
                 #[cfg(feature = "multi_template")]
                 Instruction::CallBlock(name) => {
                     if parent_instructions.is_none() {
```

Comment:
I wonder if we can also check for not being in `CaptureMode::Discard` here in addition to `parent_instructions.is_none()`. Blocks already have their own scope, so modifications can't leak out, and anything they render is discarded anyways. That would be a small but potentially somewhat meaningful performance improvement.